packages/react-native/React/Base/RCTTVRemoteHandler.m

```diff
@@ -475,12 +502,15 @@ - (void)panned:(UIPanGestureRecognizer *)gesture {
     CGPoint velocity = [gesture velocityInView:rootView];
 
     if (gestureState) {
-        [self sendAppleTVEvent:RCTTVRemoteEventPan withBody:@{@"state": gestureState,
-                                                              @"x": [NSNumber numberWithInt:translation.x],
-                                                              @"y": [NSNumber numberWithInt:translation.y],
-                                                              @"velocityX": [NSNumber numberWithFloat:velocity.x],
-                                                              @"velocityY": [NSNumber numberWithFloat:velocity.y],
-                                                            }];
+        NSNumber *eventKeyAction = [self eventKeyActionForGestureRecognizer:gesture];
```

Comment:
Swipe gestures also fall into this category. Since tvOS supports both press and touch gestures, I struggled to decide whether to assign an event key action to all gestures or just "Press" gestures.

Now, with your feedback, I can see how it may be better to keep the spirit of an event *key* to physical presses on the remote.

What do you think about sending the `eventKeyAction` only gestures that use `allowedPressTypes', i.e., UITapGestureRecognizer and UILongPressGestureRecognizer? And, for swipe and pan gestures, there would be no `eventKeyAction`, but the body would contain the state string.

Reply:
Sounds good to me!